ema_workbench/util/ema_logging.py

```diff
@@ -178,7 +178,7 @@ def get_rootlogger():
     return _rootlogger
 
 
-def log_to_stderr(level=None):
+def log_to_stderr(level=None, set_root_logger_levels=False):
     """
     Turn on logging and add a handler which prints to stderr
```

Comment:
I like this structure a lot. It would be a good idea to adapt the other evaluators accordingly. This would be a separate future issue.

Reply:
Thanks!

Let's finish this up in deed and then we can adapt the other evaluators this way afterwards.